db/fixtures/miq_product_features.yml

```diff
@@ -1082,6 +1086,10 @@
       :description: Open Admin UI for Infrastructure Providers
       :feature_type: control
       :identifier: ems_infra_admin_ui
+    - :name: Native Console
```

Comment:
@Kuldip-Nanda this is for infra provider, right? infra provider also has native console?
cc @agrare

Reply:
It is used in PowerVC context.